python/cudf/cudf/core/series.py

```diff
@@ -2597,6 +2597,10 @@ def is_monotonic_decreasing(self):
     def __cuda_array_interface__(self):
         return self._column.__cuda_array_interface__
 
+    def repeat(self, repeats):
```

Comment:
Can we add an `axis=None` kwarg here for compatibility reasons with Pandas. And we can assert that it's None.

Reply:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.repeat.html